.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
❗️ This (`end_of_line`) is not correct for all systems. Line endings for a project in Git should be controlled by **.gitattributes**, so this line needs to be omitted from **.editorconfig** altogether.